test/postsubmit-tests-with-pipeline-deployment.sh

```diff
@@ -0,0 +1,149 @@
+#!/bin/bash
```

Comment:
most of the code in this file is duplicated with presubmit-test.sh. 

I think the only difference between two is waiting for cloud builder to finish, and parameters passed when the argo workflow is invoked. Can we dedup the code so that only those are not shared. It would be a pain to maintain these scripts otherwise.

Reply:
@IronPan

Reply:
yes let;s not have duplicated code.

Reply:
Done, I have also separate the kubeflow deployment and pipeline into two scripts

Reply:
To summarize, I have separate the test_prepare, deploy_kubeflow, and deploy_pipeline from the presubmit test script.